docs/help-center/tim-features/reset-password.mdx

```diff
@@ -0,0 +1,64 @@
+---
+sidebar_label: Reset password
+title: Reset password
+hide_title: true
+displayed_sidebar: helpcenterSidebar
+sidebar_position: 5
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+
+<div class="hero hero--primary">
+  <div class="container">
+    <h2 class="hero__title">Reset Password</h2>
+    <p class="hero__subtitle">How to reset your password if you have forgotten it or want to change it.</p>
```

Comment:
Do we need to tell people what are the possible reasons they might want to reset their password? Seems a bit over the top. I think 'how to reset your password' suffices.